util/types/schemas/blockchain.mol

```diff
@@ -108,3 +108,19 @@ table WitnessArgs {
     input_type:             BytesOpt,          // Type args for input
     output_type:            BytesOpt,          // Type args for output
 }
+
+/* Types for CellDB */
```

Comment:
suggest move these schema to `extensions.mol`, there is a section for `/* Types for Storage */`